Documentation/EyeTracking/EyeTracking_TargetSelection.md

```diff
@@ -175,20 +161,23 @@ Finally, we provide you with a solution that let's you treat eye-based input com
 This has three _advantages_:
 
 - You can make sure that the hologram is only reacting to the user's eye gaze.
-- This is independent from the currently active primary input. Hence, hence you can process multiple inputs at once.
+- This is independent from the currently active primary input. Hence, you can process multiple inputs at once. For example, combine fast eye targeting with hand gestures.
```

Comment:
>. For example, [](start = 109, length = 14)

once - for example, combining fast eye targeting with hand gestures

Reply:
Sounds better! Updated!